packages/proxy/src/providers/anthropic.ts

```diff
@@ -350,9 +392,12 @@ export function anthropicCompletionToOpenAICompletion(
   completion: AnthropicCompletion,
   isFunction: boolean,
   isStructuredOutput: boolean,
-): ChatCompletion {
+): OpenAIChatCompletion {
+  // TODO: will we ever have text -> thinking -> text -> tool_use, thus are we dropping tokens?
```

Comment:
good point, i would think that's the direction things are heading with the emphasis on agents. unclear if this already happens though

Reply:
yeah mostly surprised by the choice to only grab the first. need to keep an eye on user complaints on this